source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1384,14 +1386,25 @@ export abstract class DesignerPeer extends DraggableElement {
         return result;
     }
 
-    addElementsToDesignerSurface(designerSurface: HTMLElement, processChildren: boolean = false) {
```

Comment:
out of curiosity, under what circumstances would we process children (or not) prior to this change? are those scenarios still applicable?

Reply:
From what I could tell, we didn't process the children in two scenarios:

1. Initial rendering
2. When a new peer is added (not via drop)

For the initial load, the behavior is still the same because we do not pass a neighbor.

When a new peer is added, we do not pass a neighbor _unless_ the peer was added via an element action (ex: add a column).